src/Keboola/DbExtractor/Extractor/Snowflake.php

```diff
@@ -40,6 +40,7 @@ class Snowflake extends Extractor
     private $database;
 
     /** @var string */
+
```

Comment:
Pryč s tím ;)

Reply:
je to pryč + jsem upravil i ostatní definice aby byly jednotné